config/versiondef.h.in

```diff
@@ -10,4 +10,7 @@
 #define NVIM_VERSION_CFLAGS "@NVIM_VERSION_CFLAGS@"
 #define NVIM_VERSION_BUILD_TYPE "@NVIM_VERSION_BUILD_TYPE@"
 
+#define NVIM_API_CURRENT 1
```

Comment:
Probably overkill, but what about moving this to e.g. `nvim/api/version.h`? Or maybe just move it to `nvim/version.h`? It's not generated from CMake like all the other stuff in this file, so I think it doesn't really fit.